File: javaScript/condicionales ifelse.js
// Solicitar la calificación del estudiante
let calificacion = parseFloat(prompt("Ingrese la calificación del estudiante:"));

// Evaluar la calificación y mostrar el resultado
if (calificacion >= 90) {
    console.log("Calificación Excelente: E.");
} else if (calificacion >= 80) {
    console.log("Calificación Sobresaliente: S.");
} else if (calificacion >= 60) {
    console.log("Calificación Aceptable: A.");
} else if (calificacion >= 40) {
    console.log("Calificación Insuficiente: I.");
} else {
    console.log("Calificación Deficiente: D.");
}

File: python/condicionales if else.py
# un programa para verificar si un numero es positivo o negativo

num = float(input("Ingrese un numero: "))

if num > 0 :
    print("El numero es positivo.")
elif num < 0 :
    print("El numero es negativos.")
else:
    print("el numero es cero.")
    
#calcular la calificacion de un estudiante

calificacion = float(input("Ingrese la caliificacion del estudiante: "))

if calificacion >= 90:
    print("Calificacion Excelente: E.")
elif calificacion >= 80:
    print ("Calificaion es Sobresaliente: S.")
elif calificacion >=60:
    print("Calificacion es aceptable: A.")
elif calificacion >=40:
    print("Calificacion es Insuficiente: I.")
else: 
    print("Calificacion Defisiente: D.")